Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusInterface.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of
+ * this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Serialization/SerializeContext.h>
+
+#include <AzToolsFramework/Prefab/Instance/Instance.h>
+#include <AzToolsFramework/Prefab/Template/Template.h>
+
+namespace AzToolsFramework::Prefab
+{
+    typedef AZ::Outcome<void, AZStd::string> PrefabFocusOperationResult;
+
+    /*!
+     * PrefabFocusInterface
+     */
+    class PrefabFocusInterface
+    {
+    public:
+        AZ_RTTI(PrefabFocusInterface, "{F3CFA37B-5FD8-436A-9C30-60EB54E350E1}");
+
+        //! Set the focused prefab instance to the owning instance of the entityId provided
```

Comment:
nit: missing `.`

Reply:
suggestion: Could we expand the description here slightly, I don't fully understand what is meant by the 'owning instance of the entity id'

Reply:
This class is part of the Prefab system, so it's using the prefab jargon. All entities are owned by a prefab instance, which is what the system will store internally.

Reply:
Ah okay, lol adding that description to the comment might help 😛